src/delegate-proxy/templates/_helpers.tpl

```diff
@@ -63,4 +63,16 @@ Create the name of the service account to use
 
 {{- define "delegate-proxy.pullSecrets" -}}
 {{ include "common.images.pullSecrets" (dict "images" (list .Values.image) "global" .Values.global ) }}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Create the name of the immutable delegate image to use
+*/}}
+{{- define "delegate-proxy.docker_image" -}}
+{{- $tag := printf "%s" .Values.image.tag }}
+{{- if .Values.global.useMinimalDelegateImage }}
+{{- $tag = printf "%s_minimal" $tag }}
```

Comment:
why are we using `_` in the tag. Usually it should either be `.` (dot) or `-` (dash)
I see `delegate` is using `.` (dot), so I would recommend we stick to that

Reply:
yeah, we should change that, we are the owners of publishing delegate-proxy image so we should change the convention there

Reply:
Okay, if we change the convention from newer images, I can add a check in the script based on the version provided, to use either `_` or `.` based on the version, as I suppose the previous versioned images would still use the old convention.

Reply:
Got it, sounds good.